packages/h5p-server/src/index.ts

```diff
@@ -85,6 +88,7 @@ export {
     streamToString,
     ContentFileScanner,
     ContentTypeCache,
+    ContentUserDataManager,
```

Comment:
Is it necessary to export it? The other managers are exported as they are used by the html exporter package, but I'm not sure if anyone might want to use the ContentUserDataManager outside of the library. If we export too many classes, it becomes more difficult to change signatures.

Reply:
If you want to allow users to integrate it under custom routes then yes. It might be necessary to put custom logic into a route and therefore it would be good if you can import the manager and handle the state saving on your own.